### PR TITLE
add clearer error message when checkUpdated failed

### PR DESCRIPTION
**Pull request by hanxiaoshuai**
add clearer error message when checkUpdated failed

### CODE REVIEW

File: pilot/cmd/pilot-agent/status/ready/probe.go [hunk 1]
```diff
@@ -75,5 +75,5 @@ func (p *Probe) checkUpdated() error {
 		return nil
 	}
 
-	return errors.New(s.String())
+	return errors.New("wanted cds updates > 0 and lds updates > 0, got "+s.String())
```
Comment (hzxuzhonghu) [inline pilot/cmd/pilot-agent/status/ready/probe.go]:
```suggestion
	return errors.New("proxy has not been updated")
```

Comment (hanxiaoshuai) [reply pilot/cmd/pilot-agent/status/ready/probe.go]: Done, Thanks for your review

**Pull request closed without merging**